Apps/phpanorama/panorama.json

```diff
@@ -5,14 +5,14 @@
     "publisher": "Splunk",
     "type": "firewall",
     "main_module": "panorama_connector.py",
-    "app_version": "3.0.0",
+    "app_version": "3.1.0",
```

Comment:
Could you confirm if this version update look right? I was thinking of updating this to `3.0.1`

Reply:
This CL is expecting to be using the latest asset https://cd.splunkdev.com/phantom/assets/-/merge_requests/418/diffs

Reply:
There is no need to update the app_version manually, I confirmed with Ricardo that the Jenkins job handles it.